### ISSUE TITLE
Is the addition of the 'OPTforSequenceClassification' class scheduled?

### ISSUE THREAD

**Issue by penpaperkeycode** [huggingface/transformers#17525]
### Feature request

Is the addition of the 'OPTforSequenceClassification' class scheduled? 
Is someone handling it? 
When adding these functions, I wonder if it is possible to PR one by one, or if I have to PR all classes supported by other models.

### Motivation

Added function of OPT class, which is being actively discussed recently

### Your contribution

I personally use the forSequenceClassification class because I need it for my experiments, but I would like to inquire if I can contribute to the addition of the function.

**Comment by NielsRogge** [comment]
Don't think anyone is working on this yet (cc @younesbelkada), so feel free to contribute

**Comment by ArthurZucker** [comment]
Hey! We are not working on that as Niels mentioned 😉 
Opening a PR for each framework (`pytorch`, `tensorflow` and `flax`) is recommended. If you have any question about adding these models, feel free to reach out!

**Comment by younesbelkada** [comment]
Hi, yes you can definitely go ahead and we'll be happy to review it ! ;)

**Comment by VijayKalmath** [comment]
@penpaperkeycode  as @younesbelkada  mentioned , can you please let me know if you are already working on the issue ?

**Comment by penpaperkeycode** [comment]
@VijayKalmath Sorry for late reply.
I tested OPT's 'forSequenceClassification' class and verified accuracy in MNLI-glue task. 
it looks good to me. But since this is my first huggingface PR, I need to figure out how to do it.

In addition, I am working on a conditional generation class, but there is no other work other than these two classes.

**Comment by skanjila** [comment]
@younesbelkada @ArthurZucker I am new to the hugging face community and am looking to contribute, I looked through the first issue list but all issues seem to have pre existing contributors assigned, can I help with this one or is there another first issue I can dive into and help out with.  Thanks

**Comment by ArthurZucker** [comment]
Hey, it seems that this was already taken car of, see #18123, but thanks a lot for wanting to contribute! I am going to look for an issue that might be interesting for you :) 
BTW, you can also check the closed issues as some are close because of a lack of activity. I assigned myself to #17514 but have not had the time to really look into it! Feel free to fork `transformers` and create a branch for the fix !
